### PR TITLE
Fix linting errors so CircleCI can pass

### PR DESCRIPTION
**Pull request by doub1ejack**
image

If #277 looks good, I'd love to merge that in also so these issues that (presumably) keep stalling out CircleCI can be avoided. That would keep the QA builds for Expo/Testflight more up to date and help out testing.

### CODE REVIEW

File: components/disposal-site-selector/disposal-site-selector.js [hunk 1]
```diff
@@ -1,29 +1,17 @@
 // @flow
 import React, { Fragment, useEffect, useState } from "react";
-import { View, StyleSheet, Modal } from "react-native";
+import { Modal } from "react-native";
```
Comment (doub1ejack) [inline components/disposal-site-selector/disposal-site-selector.js:3]: image

File: components/search-bar/search-bar.js [hunk 2]
```diff
@@ -4,7 +4,6 @@ import { StyleSheet, View, TextInput, TouchableHighlight, Platform } from "react
 import { Ionicons } from "@expo/vector-icons";
 import { defaultStyles } from "../../styles/default-styles";
 import { Button, Lightbox } from "@shoutem/ui";
-import TrashInfo from "../trash-info/trash-info";
 
 const localStyles = {
     searchBar: {
```
Comment (doub1ejack) [inline components/search-bar/search-bar.js:9]: image

File: components/town-disposal-details/town-disposal-details.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React, { Fragment } from "react";
 import { defaultStyles } from "../../styles/default-styles";
 import Address from "../../models/address";
 import { StyleSheet, ScrollView, View, SafeAreaView, Platform } from "react-native";
-import { Subtitle, Text, Title, Divider, Caption } from "@shoutem/ui";
+import { Subtitle, Text, Title, Divider } from "@shoutem/ui";
 import moment from "moment";
 import MiniMap from "../mini-map";
 import ButtonBar from "../button-bar/";
```
Comment (doub1ejack) [inline components/town-disposal-details/town-disposal-details.js:9]: image

File: components/town-list-item/town-list-item.js [hunk 4]
```diff
@@ -2,39 +2,35 @@
...
+        } }>
+            { town.townName || "" }
+        </Text>
+        <SimpleLineIcons
+            name={ "arrow-right" }
+            size={ 20 }
+            color="#333"
+        />
+    </TouchableOpacity>
+);
```
Comment (doub1ejack) [inline components/town-list-item/town-list-item.js:36]: image

File: components/trash-drop-form/trash-drop-form.js [hunk 5]
```diff
@@ -87,23 +87,29 @@ export const TrashDropForm = ({ location, trashDrop, onSave, currentUser, townDa
...
-        if (townData[townKey].townId === selectedSite.townId) {
-            selectedTown = townData[townKey];
-            return true;
-        }
+        const foundSelectedTown = townData[townKey].townId === selectedSite.townId
+            ? true
+            : false;
+        if(foundSelectedTown) { selectedTown = townData[townKey]; }
+        return foundSelectedTown;
     });
```
Comment (doub1ejack) [inline components/trash-drop-form/trash-drop-form.js:113]: image

**Review by doub1ejack — COMMENTED** [review]
basic linting errors

**Review by iritush — APPROVED** [review]
Looks good

**Pull request merged**